packages/backend-lib/src/server/genericErrorMiddleware.ts

```diff
@@ -1,10 +1,9 @@
 import type { AppError, BackendErrorResponseObject, ErrorObject } from '@naturalcycles/js-lib'
 import { _anyToError, _errorLikeToErrorObject, _filterUndefinedValues } from '@naturalcycles/js-lib'
-import type { SentrySharedService } from '../sentry/sentry.shared.service.js'
 import type { BackendErrorRequestHandler, BackendRequest, BackendResponse } from './server.model.js'
 
 export interface GenericErrorMiddlewareCfg {
-  sentryService?: SentrySharedService
+  errorService?: ErrorReportingService
```

Comment:
I propose longer and more descriptive name here, as `errorService` is not as descriptive and clear for a high-level interface 🙏 

```suggestion
  errorReportingService?: ErrorReportingService
```